src/index.js

```diff
@@ -741,6 +763,11 @@ import { isRemote } from 'pouchdb-utils';
                             db = databases[databaseParam];
                         }
                         if (!db) {
+                            vm.$emit('pouchdb-livefeed-error', {
+                                db: key,
+                                config: config,
```

Comment:
not sure what config is good for in this error

Reply:
good point, i'll keep it simpler